test/integration/bad/missing_dist_suffix/stanc.expected

```diff
@@ -58,9 +79,42 @@ Semantic error in 'non_existing_distribution_suffix4.stan', line 4, column 14 to
    -------------------------------------------------
      2:  }
      3:  model {  
-     4:      target += von_mises_notasuffix(1|0,1);
+     4:      target += von_mises_notasuffix(1,0,1);
+                       ^
+     5:  }
+   -------------------------------------------------
+
+A returning function was expected but an undeclared identifier 'von_mises_notasuffix' was supplied.
+  $ ../../../../../install/default/bin/stanc user_defined.stan
+Semantic error in 'user_defined.stan', line 7, column 13 to column 27:
+   -------------------------------------------------
+     5:  }
+     6:  model {
+     7:     target += bar_lpmf(19.2);
+                      ^
+     8:  }
+   -------------------------------------------------
+
+Function 'bar_lpmf' is not implemented for distribution 'bar', use 'bar_lpdf' instead.
```

Comment:
It's interesting to me that this one is caught but the other user defined functions are not. What happens if you test this same model but without the functions block, just the model block. Does it still tell you to use the (now nonexistant) `bar_lpdf`?